config/src/main/scala/bloop/config/Config.scala

```diff
@@ -236,6 +236,12 @@ object Config {
       excludes: List[String]
   )
 
+  case class SourceGenerator(
+      sourcesGlobs: List[SourcesGlobs],
+      outputDirectory: Path,
+      argv: List[String]
```

Comment:
You're right that it's not the most explicit... I'll change it for `command` since this represent the entire command line, not just the arguments you would pass to a command.

Reply:
That's sounds better!